extensions/wikia/AnalyticsEngine/js/universal_analytics.js

```diff
@@ -61,6 +61,11 @@
 				trackName: 'pageFair',
 				dimension: 7,
 				name: 'pagefair'
+			},
+			babDetector: {
+				trackName: 'babDetector',
+				dimension: 30,
```

Comment:
Can we use `6` and remove source point? It's more safe as we don't know if someone is using `30`.

Reply:
Done.